data/discovery/registry/national-health-service.yaml

```diff
@@ -0,0 +1 @@
+registry: national-health-service
```

Comment:
This doesn't seem to match any of the `public-bodies` we have. I believe we need to use one that matches, otherwise you won't get a crest showing up.